kubeportal/api/views/groups.py

```diff
@@ -0,0 +1,24 @@
+from drf_spectacular.types import OpenApiTypes
+from drf_spectacular.utils import extend_schema_view, extend_schema, OpenApiParameter
+from rest_framework import serializers, viewsets
+
+from kubeportal.models.portalgroup import PortalGroup
+
+
+class GroupSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = PortalGroup
+        fields = ('name',)
```

Comment:
why do you use a tuple? why not ['name']? is it a django feature? does it matter?

Reply:
I really don't know. It just works ...